HerPublicWebsite.UnitTests/BusinessLogic/Services/CsvFileCreatorTests.cs

```diff
@@ -31,7 +31,7 @@ public void CreateReferralRequestFileData_CalledWithReferralRequest_GeneratesExp
         var reader = new StreamReader(data, Encoding.UTF8);
         reader.ReadToEnd().Should().Be(
 "Referral date,Referral code,Name,Email,Telephone,Address1,Address2,Town,County,Postcode,UPRN,EPC Band,EPC confirmed by homeowner,EPC Lodgement Date,Is off gas grid,Household income band,Is eligible postcode,Tenure\r\n" +
-"2023-01-01 13:00:01,DummyCode00001,Full Name1,contact1@example.com,00001 123456,Address 1 line 1,Address 1 line 2,Town1,County1,AL01 1RS,100 111 222 001,E,,2023-01-01 15:00:01,yes,Below £31k,no,Owner\r\n");
+"2023-01-01 13:00:01,DummyCode00001,Full Name1,contact1@example.com,00001 123456,Address 1 line 1,Address 1 line 2,Town1,County1,AL01 1RS,100 111 222 001,E,,2023-01-01 15:00:01,yes,\"£36,000 or less\",no,Owner\r\n");
```

Comment:
Why are we getting quote marks here now?

Reply:
The new approved copy text has a comma in it, which means it needs quote marks in the CSV to escape that comma.